tox.ini

```diff
@@ -4,14 +4,13 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py278, py27, py33, py34, py35
+envlist = riak-py278, riak-py27, riak-py33, riak-py34, riak-py35
 
-[testenv:py278]
+[testenv:riak-py278]
 basepython = {env:HOME}/.pyenv/versions/riak-py278/bin/python2.7
 
 [testenv]
 install_command = pip install --upgrade {packages}
 commands = {envpython} setup.py test
-deps = six
```

Comment:
In a clean env, listing `six` here would hide the dependency issue in the riak-python-client code as well as basho-erlastic.